bitrise.yml

```diff
@@ -2938,8 +2955,8 @@ workflows:
     meta:
       bitrise.io:
         stack: osx-xcode-10.3.x
-    after_run:
-    - RunSmokeXCUITests
+    #after_run:
+    #- RunSmokeXCUITests
```

Comment:
Not sure if adding the run_if here will work.
Let's comment this for now until we have the environment set as working, running unit tests would be enough.